test/e2e/parallel/proxy_test.go

```diff
@@ -969,6 +984,24 @@ func TestSpaceLister(t *testing.T) {
 		require.NoError(t, err)
 	})
 
+	t.Run("banned user cannot get workspace", func(t *testing.T) {
+		// when
+		workspace, err := users["banneduser"].getWorkspace(t, hostAwait, users["banneduser"].compliantUsername)
+
+		// then
+		require.EqualError(t, err, "the server could not find the requested resource (get workspaces.toolchain.dev.openshift.com banneduser)")
```

Comment:
Is it correct that it's returning an error on banneduser? I'd expect either a Forbidden or a NoMatch on workspaces

Reply:
actually it looks like there should be an error returned when the proxy is initialized, as pointed out https://github.com/codeready-toolchain/toolchain-e2e/pull/1037#discussion_r1723049502 . 

I've updated the test to only try and initialize the proxy client for the banned user, which will result in an error.